CHANGELOG.md

```diff
@@ -3,6 +3,329 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.9
+
+_Jan 18, 2024_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Allow to filter non-filterable columns programmatically (#11538) @MBilalShafi
+- 🚀 Uplift `SimpleTreeView` customization examples (#11424) @noraleonte
+- 🌍 Add Croatian (hr-HR), Portuguese (pt-PT), and Chinese (Hong Kong) (zh-HK) locales (#11668) on the Data Grid @BCaspari
+- 🐞 Bugfixes
+
+### Data Grid
+
+#### Breaking changes
+
+- Non-filterable columns could now be filtered programmatically by controlling or initialing `filterModel`, or by updating the `filterModel` by API method `apiRef.current.setFilterModel`.
+
+- The `ariaV7` experimental flag has been removed and the Data Grid now uses the improved accessibility implementation by default.
```

Comment:
Definitely worth linking the said "improved accessibility" stuff here.

Reply:
@cherniavskii do we have (a collection of) links for this?

Reply:
There is a list of changes below